ui/tests/acceptance/secrets/backend/database/workflow-test.js

```diff
@@ -0,0 +1,336 @@
+/**
+ * Copyright (c) HashiCorp, Inc.
+ * SPDX-License-Identifier: BUSL-1.1
+ */
+
+import { module, test } from 'qunit';
+import { v4 as uuidv4 } from 'uuid';
+import { Response } from 'miragejs';
+import { click, currentURL, fillIn, visit } from '@ember/test-helpers';
+import { setupMirage } from 'ember-cli-mirage/test-support';
+import { create } from 'ember-cli-page-object';
+
+import ENV from 'vault/config/environment';
+import { setupApplicationTest } from 'vault/tests/helpers';
+import authPage from 'vault/tests/pages/auth';
+import flashMessage from 'vault/tests/pages/components/flash-message';
+import { deleteEngineCmd, mountEngineCmd, runCmd } from 'vault/tests/helpers/commands';
+
+const flash = create(flashMessage);
+
+const PAGE = {
+  // GENERIC
```

Comment:
There's a `helpers/general-selectors` file some of these generic selectors could be imported from. I've been adding any missing selectors to that file as I go

Reply:
I understand you may not want to use that here for backporting! So mostly just letting you know

Reply:
There are some follow-on items I want to do for this engine so I'll move the selectors once we're clear from backporting